src/dmd/expressionsem.d

```diff
@@ -2221,6 +2221,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
                     return setError();
 
                 exp.checkDeprecated(sc, f);
+                exp.checkDisabled(sc, f);
                 exp.checkPurity(sc, f);
                 exp.checkSafety(sc, f);
                 exp.checkNogc(sc, f);
```

Comment:
Looking at the diff, it seems like these five function calls are a good candidate to group together in a `doChecks` function? (not required for this PR)

Reply:
-> https://github.com/dlang/dmd/pull/7628